packages/components/breadcrumb/_breadcrumb.scss

```diff
@@ -16,10 +16,8 @@
 
 .nhsuk-breadcrumb {
   @include print-hide();
-
-  background-color: $color_nhsuk-white;
-  padding-bottom: 12px; /* [1] */
-  padding-top: 12px; /* [1] */
+  padding-bottom: nhsuk-spacing(3);
+  padding-top: 20px;
```

Comment:
Is there any possibility we can stick to the Spacing scale for this padding-top? https://service-manual.nhs.uk/design-system/styles/spacing

If not, not a major worry but we try stick to the scale as much as we can.